gulpfile.ts

```diff
@@ -159,6 +160,22 @@ function writeFilePromise(path, content) {
   });
 }
 
+function sitemap(cards: SingleCardJSON[]): SingleCardJSON[] {
+  const baseurl = 'http://www.aliemcards.com';
+  let sitemap = `${baseurl}\n`;
```

Comment:
The template literal caused ident problems. Stole your idea with map().join() and just built a single array for the sitemap.